recipes/r-seurat-scripts/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = '0.0.1' %}
+
+package:
+  name: r-seurat-scripts
+  version: {{ version }}
+
+source:
+  url: https://github.com/ebi-gene-expression-group/r-seurat-scripts/archive/v{{ version }}.tar.gz
+  sha256: 580770b74194b41656e39af542af01250326f41a311038249b84dede8a57f76c
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+    run:
+        - r-seurat>=2.3.1
+        - r-optparse
+        - r-workflowscriptscommon>=0.0.1
+
+test:
+    commands:
+        - which seurat-read-10x.R
```

Comment:
A `seurat-read-10x.R --help` or `--version` would be nice. As this would also check the R imports.